x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -503,43 +503,62 @@ export const getEndpointConsoleCommands = ({
 
   switch (agentType) {
     case 'sentinel_one':
-      return adjustCommandsForSentinelOne({ commandList: consoleCommands });
+      return adjustCommandsForVendor({
+        commandList: consoleCommands,
+        vendor: 'sentinel_one',
+        featureFlags: {
```

Comment:
(optional) Consider NOT passing the feature flags down to the adjustment function - there is no need for it. That function can directly access the feature flags from `ExperimentalFeaturesService.get()`. Keeping it as an arg will also increase the amount of refactoring we will need to do when these are removed.

Reply:
Thanks! I see your point 👍 I used it this way so I won't have to get SentinelOne and Crowdstrike feature flags all together in  `adjustCommandsForVendor`, but since I am going to revert the logic and create 2 separate adjustCommands functions - I am going to use ExperimentalFeaturesService.get() inside 👍